EPRN.Portal/Views/Waste/Note.cshtml

```diff
@@ -19,7 +22,18 @@
                 </label>
             </h1>
             <div id="with-hint-hint" class="govuk-hint">
-                @NoteResources.HintText
+                @switch (Model.WasteCategory)
+                {
+                    case Category.Reprocessor:
+                        @NoteResources.HintTextWithReplace.Replace("<PRNTYPE>", "PRN");
+                        break;
+                    case Category.Exporter:
+                        @NoteResources.HintTextWithReplace.Replace("<PRNTYPE>", "PERN");
+                        break;
+                    case Category.Unknown:
+                        @NoteResources.HintTextWithReplace.Replace("<PRNTYPE>", "<UNKNOWN WASTE TYPE>");
+                        break;
+                }              
```

Comment:
It never even crossed my mind that the acronym PRN/PERN could be language specific. Instead of hardcoding the replacement values, I can add PRN and PERN as entries in the MasterResource file, and pick it up from there. What do you think?